src/service-override/extensionGallery.ts

```diff
@@ -0,0 +1,55 @@
+import { ExtensionManagementServerService } from 'vs/workbench/services/extensionManagement/common/extensionManagementServerService'
+import { IExtensionIgnoredRecommendationsService, IExtensionRecommendationsService } from 'vs/workbench/services/extensionRecommendations/common/extensionRecommendations'
+import { ExtensionRecommendationsService } from 'vs/workbench/contrib/extensions/browser/extensionRecommendationsService'
+import { WebExtensionsScannerService } from 'vs/workbench/services/extensionManagement/browser/webExtensionsScannerService'
+import { IExtensionRecommendationNotificationService } from 'vs/platform/extensionRecommendations/common/extensionRecommendations'
+import { IIgnoredExtensionsManagementService, IgnoredExtensionsManagementService } from 'vs/platform/userDataSync/common/ignoredExtensions'
+import { ExtensionManifestPropertiesService, IExtensionManifestPropertiesService } from 'vs/workbench/services/extensions/common/extensionManifestPropertiesService'
+import { IBuiltinExtensionsScannerService } from 'vs/platform/extensions/common/extensions'
+import { BuiltinExtensionsScannerService } from 'vs/workbench/services/extensionManagement/browser/builtinExtensionsScannerService'
+import { ExtensionIgnoredRecommendationsService } from 'vs/workbench/services/extensionRecommendations/common/extensionIgnoredRecommendationsService'
+import { IWorkspaceExtensionsConfigService, WorkspaceExtensionsConfigService } from 'vs/workbench/services/extensionRecommendations/common/workspaceExtensionsConfig'
+import { IRemoteExtensionsScannerService } from 'vs/platform/remote/common/remoteExtensionsScanner'
+import { RemoteExtensionsScannerService } from 'vs/workbench/services/remote/common/remoteExtensionsScanner'
+import { ExtensionRecommendationNotificationService } from 'vs/workbench/contrib/extensions/browser/extensionRecommendationNotificationService'
+import { ExtensionTipsService } from 'vs/platform/extensionManagement/common/extensionTipsService'
+import { ExtensionManagementService } from 'vs/workbench/services/extensionManagement/common/extensionManagementService'
+import 'vs/workbench/contrib/extensions/browser/extensions.contribution'
+import 'vs/workbench/contrib/extensions/browser/extensions.web.contribution'
+import 'vs/workbench/contrib/logs/common/logs.contribution'
+import { IRemoteUserDataProfilesService, RemoteUserDataProfilesService } from 'vs/workbench/services/userDataProfile/common/remoteUserDataProfiles'
+import { ExtensionEnablementService } from 'vs/workbench/services/extensionManagement/browser/extensionEnablementService'
+import 'vs/workbench/services/extensionManagement/browser/extensionBisect'
+
+export default function getServiceOverride (): IEditorOverrideServices {
+  return {
+    [ILocaleService.toString()]: new SyncDescriptor(WebLocaleService, [], false),
```

Comment:
Do we need it? it seems to only set and clear localStorage values we don't use

Reply:
We don't need it!